.github/workflows/test.yml

```diff
@@ -70,16 +70,16 @@ jobs:
       fail-fast: false
       matrix:
         elasticsearch:
-          - '7.11.2'
-          - '7.12.1'
-          - '7.13.4'
```

Comment:
7.13 has not yet been made [EOL](https://www.elastic.co/support/eol). Can we leave that one in please.

Reply:
Ah! restored 🙏
https://github.com/elastic/terraform-provider-elasticstack/pull/230/commits/124b59b14e0e82a803ba281759db66515a410ee9